README.md

```diff
@@ -6,9 +6,12 @@
 [![License](https://img.shields.io/github/license/DeterminateSystems/nix-installer)](https://github.com/DeterminateSystems/nix-installer/blob/main/LICENSE)
 [![Discord](https://img.shields.io/discord/1116012109709463613)](https://determinate.systems/discord)
 
-**Determinate Nix Installer** is the best way to install [Nix].
-It works on macOS, Linux, Windows Subsystem for Linux (WSL), SELinux, the Valve Steam Deck, and more.
-It installs either [Nix](https://nixos.org) or [Determinate Nix][det-nix] (with [flakes] enabled by default), it offers support for seamlessly [uninstalling Nix](#uninstalling), it enables Nix to survive [macOS upgrades][macos-upgrades], and [much more](#features).
+**Determinate Nix Installer** is the easiest and most reliable way to install [Nix]—and it's also our longest-running project.
+
+When we created Determinate Nix, we added support for it directly into the installer via the ```--determinate``` flag.
```

Comment:
Inside of sentences, you only need a single backtick for code snippets. The triple backticks are for multi-line code snippets.